src/modelinstance.hpp

```diff
@@ -252,13 +252,17 @@ class ModelInstance {
          */
     std::unique_ptr<OVInferRequestsQueue> inferRequestsQueue;
 
+    ov::AnyMap rt_info;
```

Comment:
We have multiple places containing rt_info now. One inside the ov::Model and one inside ModelInstance. Can't we just pass ov::Model RT info in ModelInstance::GetRTInfo call?

Reply:
There you extract rt_info from input, here directly from ov::Model.